### PR TITLE
Return arg in case of invalid param in strftime

### PR DESCRIPTION
**Pull request by itsankitkp**
Chrono panics in case of unsupported formats, this patch handles such cases and returns supplied format as a result.

### CODE REVIEW

File: extra_tests/snippets/stdlib_datetime.py [hunk 1]
```diff
@@ -131,6 +131,11 @@ def __init__(self, offset, name):
 assert_raises(NotImplementedError, ne.utcoffset, dt)
 assert_raises(NotImplementedError, ne.dst, dt)
 
+# unsupport format in strptime returns arg itself
+# in linux. Todo: add cases for Mac/Windows
+if sys.platform in ('linux'):
```
[the remark this answers is not in the document]
Comment (itsankitkp) [reply extra_tests/snippets/stdlib_datetime.py]: > `('linux')` is equivalent to `'linux'`. The original code tests if 'linux' contains sys.platform

I have fixed this

[the remark this answers is not in the document]
Comment (itsankitkp) [reply extra_tests/snippets/stdlib_datetime.py]: > you don't need to open a new pr to fix branch. please ask help next time. I can help you to fix them \
Sure, next time I will do (messing up git history is not that unusual 😄 )

File: extra_tests/snippets/stdlib_datetime.py [hunk 2]
```diff
@@ -131,6 +131,11 @@ def __init__(self, offset, name):
 assert_raises(NotImplementedError, ne.utcoffset, dt)
 assert_raises(NotImplementedError, ne.dst, dt)
 
+# unsupport format in strptime returns arg itself
+# in linux. Todo: add cases for Mac/Windows
+if sys.platform == 'linux':
```
Comment (youknowone) [inline extra_tests/snippets/stdlib_datetime.py:136]:
```suggestion
if sys.platform.startswith("linux"):
```
I found `linux` is not the exact name but prefix

**Review by youknowone — APPROVED** [review]
Thank you for contributing!

**Comment by youknowone** [comment]
https://github.com/RustPython/RustPython/pull/4537/files#diff-d925268c057737ba320c489ca302fd5125f14840d22e50cfe226bdb5bb88ba8dR546-R553

**Pull request merged**